src/util/globals.js

```diff
@@ -33,6 +33,8 @@ export const task = (function () {
  * Only use functions starting with "get", as other functions are used by the current grids logic.
  */
 export let globalGridData = { data: undefined };
+//^^^actually, why is this even global...?
```

Comment:
something only you can answer, since you seem to have added it :1st_place_medal:

Reply:
I actually did remove it in askCount branch already, so it was just to remind myself.